iMX8Pkg/iMX8ConfigDsc.inc

```diff
@@ -1,155 +0,0 @@
-#
-# iMX8 Common Configurations Definition
-#
-#  Copyright (c) 2018, Microsoft Corporation. All rights reserved.
-#
-#  This program and the accompanying materials
-#  are licensed and made available under the terms and conditions of the BSD License
-#  which accompanies this distribution.  The full text of the license may be found at
-#  http://opensource.org/licenses/bsd-license.php
-#
-#  THE PROGRAM IS DISTRIBUTED UNDER THE BSD LICENSE ON AN "AS IS" BASIS,
-#  WITHOUT WARRANTIES OR REPRESENTATIONS OF ANY KIND, EITHER EXPRESS OR IMPLIED.
-#
-
-[Defines]
-
-  # Display surface will be drawn to.
-!ifndef CONFIG_HEADLESS
-  DEFINE CONFIG_HEADLESS = FALSE
-!endif
-
-  # Useful if working with Lauterbach JTAG. Will dump to console the complete
-  # command to copy and past into the Lauterbach command line to load symbols
-  # for each module.
-  # Use in conjunction with Informational debug messages:
-  # DEBUG_INFO      0x00000040
-!ifndef CONFIG_DUMP_SYMBOL_INFO
-  DEFINE CONFIG_DUMP_SYMBOL_INFO = TRUE
-!endif
-
-  # Enable if PSCI is implemented.
-!ifndef CONFIG_MPCORE
-  DEFINE CONFIG_MPCORE = TRUE
-!endif
-
-  # Disable the USB stack by default. It currently has issues and need to be
-  # taken care of
-!ifndef CONFIG_USB
-  DEFINE CONFIG_USB = FALSE
-!endif
-
-  # Disable the PCIexpress stack by default. Enable on demand.
-!ifndef CONFIG_PCIE
-  DEFINE CONFIG_PCIE = FALSE
-!endif
-
-  # States whether OPTEE boot flow is in effect or not. This has the following
-  # implications:
-  # - OPTEE must have been loaded by ATF.
-  # - Specific memory layout that is defined in the platform .dsc file.
-!ifndef CONFIG_OPTEE
-  DEFINE CONFIG_OPTEE = TRUE
-!endif
-
-  # Allow collecting performance tracing from OPTEE hot code paths
-  # Performance summary results can be inspected with the shell app Dp.efi
-!ifndef CONFIG_OPTEE_PROFILE
-  DEFINE CONFIG_OPTEE_PROFILE = FALSE
-!endif
-
-  # Call build with -D CONFIG_NOT_SECURE_UEFI=1 to disable the following security features
-!if $(CONFIG_NOT_SECURE_UEFI) == TRUE
-
-  !ifndef CONFIG_AUTH_VAR
-    DEFINE CONFIG_AUTH_VAR = FALSE
-  !endif
-
-  !ifndef CONFIG_MEASURED_BOOT
-    DEFINE CONFIG_MEASURED_BOOT = FALSE
-  !endif
-
-  !ifndef CONFIG_SECURE_BOOT
-    DEFINE CONFIG_SECURE_BOOT = FALSE
-  !endif
-
-!else
-
-  # UEFI authenticated variable runtime services
-  !ifndef CONFIG_AUTH_VAR
-    DEFINE CONFIG_AUTH_VAR = TRUE
-  !endif
-
-  # TPM stack for the fTPM
-  !ifndef CONFIG_MEASURED_BOOT
-    DEFINE CONFIG_MEASURED_BOOT = TRUE
-  !endif
-
-  # If Platform Key (PK) is provisioned, then SecureBoot variables will get
```

Comment:
We're losing these comments.  They should be moved over to the new files as relevant.

Reply:
the comments are in PlatformBuild.py